content/blogs/release-0-23.md

```diff
@@ -0,0 +1,711 @@
+---
+title: Kestra 0.23 introduces Multi-Panel Editor, No-Code Redesign, and Unit Testing
+description: Kestra 0.23 delivers a multi-panel editor, revamped no-code flow editor, YAML unit testing, new filters, and new plugins for a more productive orchestration experience.
+date: 2025-06-17T17:00:00
+category: News & Product Updates
+author:
+  name: Benoit Pimpaud
+  image: bpimpaud
+image: /blogs/release-0-23.jpg
+---
+
+
+The table below highlights the key features of this release.
+
+| Feature                                   | Description                                                                | Edition |
+|-------------------------------------------|----------------------------------------------------------------------------| --- |
+| Multi Panel Editor       | Enhanced visual editor with split-screen capabilities for simultaneous code and visual editing | All Edition |
+| No-Code Flow Editor | Create Kestra flows from the new form-based UI without writing code | All Edition |
+| Unit Test Flows       | Built-in testing framework for validating workflow behavior | Enterprise Edition |
```

Comment:
```suggestion
| Unit Tests for Flows       | With Unit Tests, we're introducing a language-agnostic, declarative interface to test your flows with fixtures and assertions, allowing you to run tests directly from the UI and catch regressions before they reach production. | Enterprise Edition |
```
it's not a framework, it's a new resource/feature in Kestra API

I'd also move this to the very top as #1 feature as it's more interesting/new than multi-panel/no-code which people need to see to udnerstand